engine/src/io/DataLoader.cpp

```diff
@@ -32,11 +32,11 @@ data_loader::~data_loader() {}
 
 
 std::unique_ptr<ral::frame::BlazingTable> data_loader::load_batch(
-	const std::vector<int> & column_indices_in,
-	const Schema & schema,
-	data_handle file_data_handle,
-	size_t file_index,
-	std::vector<cudf::size_type> row_group_ids) {
+	const std::vector<int> & /*column_indices_in*/,
```

Comment:
I think this whole class is dead code. Please delete it

Reply:
make sure it is first before deleting :-)

Reply:
It seems that only this function is deprecated. Maybe is still a valid abstraction for grouping parsers and providers.